.github/workflows/self-test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install tox
+        python -m pip install tox
```

Comment:
Unrelated?

Reply:
(at least move this to its own PR with an explanation why this is needed)

Reply:
not needed, just esthetics